src/styles/main.scss

```diff
@@ -1,3 +1,63 @@
+$cell-size: 100px;
+$gap: 1px;
+
 body {
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
   margin: 0;
 }
+
+* {
+  box-sizing: border-box;
+}
+
+.calendar {
+  gap: $gap;
+  width: 750px;
```

Comment:
The width of the calendar should be calculated based on the cell size, gap, and padding. Use the formula 'width: ($cell-size * 7) + ($gap * 6) + 10px;'